resources/views/users/partials/delete-user-form.blade.php

```diff
@@ -0,0 +1,120 @@
+<section>
+    <form id="send-verification" method="post" action="{{ route('verification.send') }}">
+        @csrf
+    </form>
+
+    <form method="post" action="{{ route('users.store') }}" class="mt-6 space-y-6">
```

Comment:
A post request to the `users.store` route? You sure this is right for delete user?

Reply:
this was caused by me moving the delete form into its own partial. i clearly really messed this up, as I *just didnt* change anything at all from the creation form so decided to just revert to when it was working.